eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -2399,6 +2444,19 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
     goto(ERR_INFORMATION_LEAK) calling doPublish(localCommitPublished, d.commitments) sending error
   }
 
+  private def handleOutdatedCommitment(channelReestablish: ChannelReestablish, d: HasCommitments) = {
+    nodeParams.outdatedCommitmentStrategy match {
+      case OutdatedCommitmentStrategy.HaltIfJustRestarted if ManagementFactory.getRuntimeMXBean.getUptime.millis < 10.minutes =>
+        log.error("we just restarted and may have an outdated commitment! stopping the node")
```

Comment:
I'd put a longer, more thorough message, something like: "we just restarted and may have an outdated commitment: the channel should force-close but the node is stopping instead, please ensure your database is up-to-date before restarting"

Reply:
Done with a25bc6a.